### PR TITLE
Feat: Added 'images', 'tasks' options and containers 'regexp_filter' in cv_facts_v3 with examples

### PR DESCRIPTION
**Pull request by ghost**
## Change Summary

Added 'images' and 'tasks' options in cv_facts_v3 with examples

## Component(s) name

`arista.cvp.cv_facts_v3`

## Proposed changes
1. Added `images` option to cv_facts_v3:
- It collects facts for images present in CVP.
- Filter by `image_name` can be applied by `regexp_filter` option, to get the specific image facts.

2. Added `tasks` option to cv_facts_v3:
- It collects facts for tasks present in CVP.
- Filter by `task_id` and `task_status` can be applied by `regexp_filter` option, to get the specific task facts.
- Verbose can be applied with `verbose` option to get long/short facts.

3. Added `regexp_filter` for `containers` option in cv_facts_v3:
- It filters the container facts by `container name` mentioned in regexp_filter argument.

## How to test

Run the playbooks in `examples/cv_facts_v3` directory 

* Follow the instruction on [Readme](https://github.com/aristanetworks/ansible-cvp/blob/devel/tests/PR_testing/README.md) to setup testing environment and follow instructions to run example playbooks

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly. (check the box if not applicable)
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Comment by sugetha24** [comment]
Can we update Documentation for this?

**Comment by Shivani-chourasiya** [comment]
> Can we update Documentation for this?

Updated the docs!

**Review by sugetha24 — APPROVED** [review]

File: ansible_collections/arista/cvp/examples/cv_facts_v3/get_configlet_facts.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    - name: Display configlet facts
+      debug:
+        msg: "{{CV_FACTS_V3_RESULT}}"
+
+    - name: Collect configlet facts from {{inventory_hostname}}
+      arista.cvp.cv_facts_v3:
+        facts:
+          - configlet
+        regexp_filter: 'cofiglet1' # collects specific configlet facts
```
Comment (sugetha24) [inline ansible_collections/arista/cvp/examples/cv_facts_v3/get_configlet_facts.yml]: i think you meant `'configlet1'` ?

Comment (Shivani-chourasiya) [reply ansible_collections/arista/cvp/examples/cv_facts_v3/get_configlet_facts.yml]: yes, its a typo..let me fix it right away

Comment (Shivani-chourasiya) [reply ansible_collections/arista/cvp/examples/cv_facts_v3/get_configlet_facts.yml]: Fixed!

**Review by sugetha24 — APPROVED** [review]

**Review by tgodaA — APPROVED** [review]
LGTM

**Pull request merged**